.ansible-lint-ignore

```diff
@@ -0,0 +1,2 @@
+# This file contains ignores rule violations for ansible-lint
+roles/systemd_timesyncd/tasks/package.yml var-naming[no-role-prefix]
```

Comment:
i don't think it is a good idea to have ignores

Reply:
I did not have the time to correct all errors at once. So i excluded these errors.